server/scala/api/src/main/scala/com/ctc/omega_edit/FFI.scala

```diff
@@ -93,6 +93,7 @@ private[omega_edit] trait FFI {
   def omega_session_get_last_undo(p: Pointer): Pointer
   def omega_session_get_num_changes(p: Pointer): Long
   def omega_session_get_checkpoint_directory(p: Pointer): String
+  def omega_session_get_checkpoint_directory_length(p: Pointer): Int
```

Comment:
Use Long here, not Int.

Reply:
Oh right because of the `int64_t` return type.